metallb/deploy_operator.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/bash
+
+METALLB_OPERATOR_REPO=${METALLB_OPERATOR_REPO:-"https://github.com/metallb/metallb-operator.git"}
+LOCAL_REGISTRY_DNS_NAME=${LOCAL_REGISTRY_DNS_NAME:-"virthost.ostest.test.metalkube.org"}
+LOCAL_REGISTRY_PORT=${LOCAL_REGISTRY_PORT:-"5000"}
+OPERATOR_IMAGE=${OPERATOR_IMAGE:-"${LOCAL_REGISTRY_DNS_NAME}:${LOCAL_REGISTRY_PORT}/metallb-operator:ci"}
+METALLB_OPERATOR_DOCKER_FILE=${METALLB_OPERATOR_DOCKER_FILE:-"Dockerfile"}
+PERSONAL_PULL_SECRET=${PERSONAL_PULL_SECRET:-"/root/dev-scripts/pull_secret.json"}
+PULL_SECRET_FILE=${PULL_SECRET_FILE:-"/opt/dev-scripts/pull_secret.json"}
+export NAMESPACE=${NAMESPACE:-"metallb-system"}
+
+if [ ! -d ./metallb-operator ]; then
+	git clone $METALLB_OPERATOR_REPO
+fi
+cd metallb-operator
+
+podman build . -f ${METALLB_OPERATOR_DOCKER_FILE} -t ${OPERATOR_IMAGE} --authfile ${PERSONAL_PULL_SECRET}
+podman push ${OPERATOR_IMAGE} --tls-verify=false --authfile ${PULL_SECRET_FILE}
+
+# install yq v4 for metallb deployment
+yq_tmp_dir=$(mktemp -d)
+cd $yq_tmp_dir
+go mod init tmp
+GO111MODULE=on go get github.com/mikefarah/yq/v4
+rm -rf $yq_tmp_dir
+cd -
+
+PATH="$GOPATH:$PATH" ENABLE_OPERATOR_WEBHOOK=true KUSTOMIZE_DEPLOY_DIR="config/webhook-on-openshift" IMG=${OPERATOR_IMAGE} make deploy
```

Comment:
do we have a way to override metallb images too? I think we'd need the latest CI gives us

Reply:
i think we will have to add a script that's setting the images like we do in the bump versions script
https://github.com/metallb/metallb-operator/blob/4d52b863b2a4764d37bf37acacf49eb2a74eee79/hack/bump_versions.sh#L6